.github/workflows/wasm-deploy.yml

```
@@ -0,0 +1,31 @@
+        uses: actions/checkout@v4
+
+      - name: Build
+        run: ./gradlew :composeApp:wasmJsBrowserDistribution
+
+      - name: Deploy
+        if: github.ref == 'refs/heads/main'
+        uses: JamesIves/github-pages-deploy-action@v4.6.3
+        with:
+          BRANCH: gh-pages
+          FOLDER: composeApp/build/dist/wasmJs/productionExecutable
+          CLEAN: true
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)